packages/field/test/FormatMixin.test.js

```diff
@@ -216,12 +235,12 @@ describe('FormatMixin', () => {
       expect(formatterSpy.callCount).to.equal(1);
 
       el.errorState = true;
-      el.modelValue = 'bar';
+      mimicUserInput(el, 'bar');
       expect(formatterSpy.callCount).to.equal(1);
-      expect(el.formattedValue).to.equal('foo: init-string');
+      expect(el.formattedValue).to.equal('bar');
```

Comment:
this should become `undefined`. Will be fixed here: https://github.com/ing-bank/lion/issues/129